src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -403,6 +407,34 @@ public SnapshotAccessRequestResponse approveRequest(UUID id) {
     return snapshotRequestDao.getById(id).toApiResponse();
   }
 
+  public SnapshotAccessRequestMembersResponse getGroupMembers(UUID id) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    return new SnapshotAccessRequestMembersResponse()
+        .members(iamService.getGroupPolicyEmails(model.samGroupName(), IamRole.MEMBER.toString()));
+  }
+
+  public SnapshotAccessRequestMembersResponse addGroupMember(UUID id, String memberEmail) {
+    SnapshotAccessRequestModel model = snapshotRequestDao.getById(id);
+    if (!ValidationUtils.isValidEmail(memberEmail)) {
+      throw new ValidationException("Invalid member email");
```

Comment:
I think ValidationException is okay, but I am wondering about the difference in usage between ValidationException and BadRequestException

Reply:
Since most `ValidationUtils` are used as `if (!isValid(x)) { throw new Exception(message); }` I think a better approach is to add a method to `ValidationUtils` to do this:
```
  public static void requireValidEmail(String email, String errorMsg) {
    if (!isValidEmail(email)) {
      throw new IllegalArgumentException(errorMsg);
    }
  }
[...]
    ValidationUtils.requireValidEmail(memberEmail, "Invalid member email");
```

Note that `IllegalArgumentException` is also mapped to HTTP 400 response, you can verify this by looking at the mappings in `GlobalExceptionHandler`